### DISCUSSION TITLE
About the upgrade procedure after the latest (v0.22.0) release (Helm-based installation)

### DISCUSSION THREAD

**Discussion by iniinikoski** [actions/actions-runner-controller#1231] [Questions] [answered]
Hi,

I've installed the ARC using Helm. Now, when the 0.22.0 is out, I'd like to perform an update.

I did see this: https://github.com/actions-runner-controller/actions-runner-controller/blob/master/charts/actions-runner-controller/docs/UPGRADING.md - and this: https://github.com/actions-runner-controller/actions-runner-controller/issues/1157#issuecomment-1055981414 - and this as well: https://github.com/actions-runner-controller/actions-runner-controller/issues/1130#issuecomment-1045763238

I also saw that the new controller-release produced an artifact in Releases: https://github.com/actions-runner-controller/actions-runner-controller/releases/tag/v0.22.0

Blondie question from here: if the Helm chart stays the same (like it did now), how can I upgrade the controller part...? I also checked the README but I could find anything regarding upgrades in general... Would it make any sense to add something next to the installation instructions?

**Comment by toast-gear** [comment, marked as answer]
~https://github.com/actions-runner-controller/actions-runner-controller/pull/1232 it'll be released in the next 10 mins~

https://github.com/actions-runner-controller/actions-runner-controller/releases/tag/actions-runner-controller-0.17.0 released

**Reply by iniinikoski** [reply]
Awesome! :)

**Reply by iniinikoski** [reply]
Another note if someone else (as a bit of a Helm noobie) hiccups on this as well at some point: remember to of course to do this before trying to upgrade to the latest version:

`helm repo update`

Thanks again @toast-gear! :)
